pkg/apis/configuration/validation/validation.go

```diff
@@ -75,8 +75,8 @@ func validateTLSRedirect(redirect *v1alpha1.TLSRedirect, fieldPath *field.Path)
 		allErrs = append(allErrs, validateTLSRedirectStatusCode(*redirect.Code, fieldPath.Child("code"))...)
 	}
 
-	if redirect.BasedOn != "scheme" && redirect.BasedOn != "x-forwarded-proto" {
-		allErrs = append(allErrs, field.Invalid(fieldPath.Child("basedOn"), redirect.BasedOn, "accepted values are 'scheme' or 'x-forwarded-proto'"))
+	if redirect.BasedOn != "" && redirect.BasedOn != "scheme" && redirect.BasedOn != "x-forwarded-proto" {
+		allErrs = append(allErrs, field.Invalid(fieldPath.Child("basedOn"), redirect.BasedOn, "accepted values are '', 'scheme', 'x-forwarded-proto'"))
```

Comment:
If the field is `''`, we assume that it is set to the default. That's why it is not necessary to report that `''` is one of the accepted value:
```suggestion
		allErrs = append(allErrs, field.Invalid(fieldPath.Child("basedOn"), redirect.BasedOn, "accepted values are 'scheme', 'x-forwarded-proto'"))
```

Reply:
sure